src/test/java/com/google/javascript/clutz/aliased_enums.d.ts

```diff
@@ -14,7 +14,7 @@ declare module 'goog:nested.bar.Enum' {
 declare namespace ಠ_ಠ.clutz_internal.nested.baz {
   type Enum = number ;
   var Enum : {
-    A : Enum ,
+    A : ಠ_ಠ.clutz_internal.nested.baz.Enum ,
```

Comment:
Yeah, I considered special casing e.g. types from the same namespace, but I think it's not worth it right now.